docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,190 @@
+ifdef::context[:parent-context: {context}]
+
+[id="prepare_director_operator_and_rhoso_for_adoption_process"]
+
+:context: migrating-databases
+
+= Preparing director operator for adoption
+
+[NOTE]
+====
+The example use case presented here is of a single OCP cluster, being shared by the source ODPdO env. _and_ a destination RHOSO env. Two OCP master nodes would be transferred to RHOSO use, leaving OSPdO with one for the adoption process, after which the remaining one can be used by RHOSO env. as well.
+====
+
+Work in the openstack project:
+[,bash]
+----
+oc project openstack
+----
+
+Define common environment variables
+[,bash]
+----
+PASSWORD_FILE="tripleo-passwords.yaml"
+export PASSWORD_FILE
+
+OSP18_NAMESPACE="osp18"
+export OSP18_NAMESPACE
+
+OSPDO_NAMESPACE="openstack"
+export OSPDO_NAMESPACE
+
+OS_CLIENT="oc exec -c openstackclient -t openstackclient -- "
+export OS_CLIENT
+
+CONTROLLER_SSH="oc rsh -n $OSPDO_NAMESPACE -c openstackclient openstackclient ssh controller-0.ctlplane"
+export CONTROLLER_SSH
+
+CONTROLLER1_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-0.ctlplane"
+export CONTROLLER1_SSH
+CONTROLLER2_SSH=
+export CONTROLLER2_SSH
+CONTROLLER3_SSH=
+export CONTROLLER3_SSH
+
+OSPDO_INTERNAL_API_NET="internalapi"
+export OSPDO_INTERNAL_API_NET
+
+STORAGE_CLASS=$(oc get -n $OSPDO_NAMESPACE pvc openstackclient-hosts -o jsonpath='{.spec.storageClassName}')
+export STORAGE_CLASS
+----
+
+Get the node name upon which the current OSP controller VM is running.  This step will need to be modified with 3 running OSP controllers.  Modify to select one of the OSP nodes.
+[,bash]
+----
+CONTROLLER_NODE=$(oc -n $OSPDO_NAMESPACE get vmi -ojson | jq -r '.items[0].status.nodeName')
+export CONTROLLER_NODE
+----
```

Comment:
```suggestion
. Get the name of the node where the current {OpenStackShort} controller VM is running:
+
----
$ CONTROLLER_NODE=$(oc -n $OSPDO_NAMESPACE get vmi -ojson | jq -r '.items[0].status.nodeName')
export CONTROLLER_NODE
----
+
Repeat this step for the remaining Controller nodes in your environment.
```
Please confirm that "Repeat this step for the remaining Controller nodes in your environment" is not changing the meaning of the original text.

Reply:
ok , and 
not sure about the question , there is no need for the "Repeat this step for the remaining .." line here , the client only needs to define the designated controller running the ODPdO controller VM.